htdocs/hrm/class/job.class.php

```diff
@@ -1071,7 +1071,7 @@ public function getKanbanView($option = '')
 		$return .= '</span>';
 		$return .= '<div class="info-box-content">';
 		$return .= '<span class="info-box-ref">'.(method_exists($this, 'getNomUrl') ? $this->getNomUrl(1) : $this->ref).'</span>';
-		$return .= '<input style="float:right;" id="cb'.$this->id.'" class="flat checkforselect" type="checkbox" name="toselect[]" value="'.$this->id.'"'.($selected ? ' checked="checked"' : '').'>';
+		$return .= '<input class="fright" id="cb'.$this->id.'" class="flat checkforselect" type="checkbox" name="toselect[]" value="'.$this->id.'"'.($selected ? ' checked="checked"' : '').'>';
```

Comment:
It seams that you added a class attribute to this input when there was already one class="flat checkforselect".
This create a bug that misplace the kanban view and make impossible to the massaction to work.
To resolve this merge the two class attributes